src/utils/Processing.ts

```diff
@@ -21,15 +22,16 @@ export function ensureOutputDir(output = path.join('.', 'gofsh')): string {
   logger.info(`Using output directory: ${output}`);
   return output;
 }
-
-export async function getResources(
-  inDir: string,
-  defs: fhirdefs.FHIRDefinitions
-): Promise<Package> {
+export async function getFhirProcessor(inDir: string, defs: fhirdefs.FHIRDefinitions) {
```

Comment:
I don't think this function is really `async`, is it?  As far as I can tell, it doesn't `await` anything and it doesn't return a `Promise` (or at least it doesn't need to).